// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

syntax = "proto2";

message simple_input {
    required int32 aggregate_counter = 1;
}
